site/docs/advanced/flood.md

```diff
@@ -25,9 +25,9 @@ This could lead to a ban.
 Moreover, as requests might be tried several times, your server will consume more RAM and bandwidth than necessary.
 Instead of fixing the problem after the fact, it is much better to enqueue all API requests and only send them at the permitted speed:
 
-## The Real Solution (recommended)
+## The Real Solution (Recommended)
```

Comment:
I agree with this, but I have a feeling that we agreed on lowercase inside parens. @roj1512 am I wrong?

Reply:
Not wrong.

Reply:
@CikiMomogi please use lowercase inside parens in all other places, too

Reply:
No, no!

Reply:
Done